### PR TITLE
Display the model name at the top of the deplyoment flow

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- use the blues flag
- start the deployment flow
- check that the model name appears at the top

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5590/
Test PASSed.

File: jujugui/static/gui/src/app/app.js [hunk 1]
```diff
@@ -938,6 +938,7 @@ YUI.add('juju-gui', function(Y) {
           listRegions={jem.listRegions.bind(jem)}
           listTemplates={jem.listTemplates.bind(jem)}
           modelCommitted={this.env.get('connected')}
+          modelName={this.db.environment.get('name')}
```
Comment (hatched) [inline jujugui/static/gui/src/app/app.js]: There is a `db` variable defined in this method.

**Comment by hatched** [comment]
Looks good thanks @huwshimi

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5596/
Test PASSed.

**Comment by anthonydillon** [comment]
Code looks good :+1: but hit an error when QA'ing:
- I added the blues flag
- I added kibana to the canvas
- Clicked on Commit changes button
- The GUI locked up and I got the following error in the console:
```
Uncaught TypeError: Cannot read property 'addTemplate' of null
```

**Comment by hatched** [comment]
This worked as expected for me. @anthonydillon it's possible that you weren't using the production JEM, all others cause issues when opening the new DF.

Thanks @huwshimi 
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**